<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesWidget</class>
 <widget class="QFrame" name="NotesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>846</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTableWidget" name="notesTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Created</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modified</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="notePlaceholder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addNoteButton">
       <property name="text">
        <string>Add note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeNoteButton">
       <property name="text">
        <string>Remove note</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="editNoteButton">
       <property name="text">
        <string>Edit note</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="hmdbCommunicationIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addNoteButton</sender>
   <signal>clicked()</signal>
   <receiver>NotesWidget</receiver>
   <slot>addNoteDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>824</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeNoteButton</sender>
   <signal>clicked()</signal>
   <receiver>NotesWidget</receiver>
   <slot>removeNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>824</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editNoteButton</sender>
   <signal>clicked()</signal>
   <receiver>NotesWidget</receiver>
   <slot>editNoteDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>824</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notesTable</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>NotesWidget</receiver>
   <slot>onCurrentNoteChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNote(int,QDateTime,QString,QString)</slot>
  <slot>addNoteDialog()</slot>
  <slot>removeNote(int)</slot>
  <slot>loadNote(int)</slot>
  <slot>clearNotes()</slot>
  <slot>clearNote()</slot>
  <slot>editNoteDialog()</slot>
  <slot>setCurrentPatient(int)</slot>
  <slot>updateNote(int,QDateTime,QString,QString)</slot>
  <slot>onCurrentNoteChange()</slot>
 </slots>
</ui>
